src/qibolab/pulses.py

```diff
@@ -169,7 +168,7 @@ def modulated_waveforms(self):
 
     def __eq__(self, item) -> bool:
         """Overloads == operator"""
-        return type(item) is self.__class__
+        return isinstance(item, self.__class__)
```

Comment:
```suggestion
        return isinstance(item, type(self))
```
Avoid using *dunders* (i.e. `__dunder__`) as much as possible, they should be considered internal mechanisms.